blueskyacademy/bluesky-nextjs
# add program page and sub navigation

## Description
- Add Program Page
- Add Sub-navigation 
- Test Logo white


## Why
- User can view the program page. There are 3 sub-page: Kindergarten, Primary, Secondary


## How to QA
- User can see a sub-navigation when click Home and About Page
- User can view program page 


## Screenshots
![Screen Shot 2021-12-31 at 09 41 47](https://user-images.githubusercontent.com/74655110/147799968-f41b987c-7638-48d9-9961-5220181b733c.png)
![Screen Shot 2021-12-31 at 09 41 57](https://user-images.githubusercontent.com/74655110/147799970-c7bda95e-9140-42a0-bb5f-e9a989d98d10.png)
